### DISCUSSION TITLE
Are CV25 SoC products supported by the SDK?

### DISCUSSION THREAD

**Discussion by sebaxis** [AxisCommunications/discussions#3] [ACAP] [answered]
Axis recently released CV25 SoC based products. Can I build ACAPs for these products? 
[Link to CV25 products in product selector](https://www.axis.com/support/tools/product-selector/shared/%5B%7B%22index%22:%5B4,0%5D,%22value%22:%5B%22CV25%22%5D%7D%5D)

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
The SDK can build binaries that work on the product, but some APIs aren't yet supported. We're currently working on officially supporting CV25, and will get back to you when we have new information on the subject.

**Reply by trevorRO** [reply]
I'm looking to use the `axoverlay` library on a camera with a CV25 SoC, is there an update on when we can expect that API to be available for that chip?  Is there a way to achieve the same results without `axoverlay`?  Am I able to draw on the video stream using other parts of the API?

**Reply by mattias-kindborg-at-work** [reply]
The ACAP SDK support for CV25 is not finished yet. When officially release we hope to provide documentation and possibly some examples.

**Reply by sebaxis** [reply]
@trevorRO: CV25 products have way less support for overlay compared to Artpec products. What information is it you want to overlay on the video? Do you necessarily want to "burn-in" the information in the video or would it be sufficient to overlay in a rendering application?

**Reply by trevorRO** [reply]
My initial steps wanted to burn-in analytics onto the frame, primary for demo purposes.  The next step, which is fine to jump to, is to crop the image based on certain detections, and only pass the cropped image.  Is that functionality currently supported on the CV25 products.

**Reply by MickSoundIntel** [reply]
@mattias-kindborg-at-work  Is there news on this specific functionality at this time, sept 2022? We develop an ACAP and currently indeed only Artpec based soc's have the overlay support. As new CV25 based camera's are being released and customers install them , this is a big one for us.

**Reply by mattias-kindborg-at-work** [reply]
We're currently working on adding support for CV25 to the ACAP SDKs, but that doesn't include the overlay functionality unfortunately.

**Reply by sebaxis** [reply]
@MickSoundIntel, what are you using the overlay functionality for? It is unlikely that we will support the same features as in ARTPEC products but may be able to provide enough for your needs.

**Reply by MickSoundIntel** [reply]
@sebaxis Our developers are utilizing the overlay function for showing a realtime spectogram, which lives in top of the live video feed.

**Comment by sebaxis** [comment]
You can get take the bounding-boxes of detection data, fetch the image and crop it in your application. Currently it is not possible the get the actual image used for the detection, but you have to fetch the current image and then assume it is quite similar to the image used moments ago from the detector. We are working on providing the detector image to applications.
